arduino/vision_runner/vision_runner.ino

```diff
@@ -15,38 +15,43 @@ double FOV_X = 75; //horizontal Field of View in degrees
 double FOV_Y = 47; //vertical Field of View in degrees
 double RESOLUTION_WIDTH = 320; //in pixels, 320 x 200
 double RESOLUTION_HEIGHT = 200; //in pixels
-double VERTICAL_ZERO_OFFSET = 15; //shifts the 0 degree line down by given value, relies on consistent angle of camera
-double CAMERA_ANGLE = 19.5; //angle of Camera
+double VERTICAL_ZERO_OFFSET = 23.5; //shifts the 0 degree line down by given value, relies on consistent angle of camera
+double CAMERA_ANGLE = 19; //angle of Camera
 
 //******DISTANCE VARIABLES******
 double distanceToTarget; //inches
-double CAMERA_HEIGHT = 39; //inches
-double TARGET_HEIGHT = 68.5; //inches
+double CAMERA_HEIGHT = 86; //inches
+double TARGET_HEIGHT = 42.5; //inches
 
 void setup() {
   Serial.begin(9600);
   pixy.init();
 }
 
 void loop() { 
-  blocks = pixy.getBlocks();
+  blocks = pixy.getBlocks(2);
+
+  if (blocks == 2) { //make sure we see two blocks
+     if(abs(pixy.blocks[0].y - pixy.blocks[1].y) > abs(pixy.blocks[0].x - pixy.blocks[1].x)){ //checks that vertical distance between blocks is greater than horizontal distance between blocks, meaning that we are seeing the boiler target
+       STATE = "BOIL";
+       updateValuesBoiler();
```

Comment:
Don't have this operate by side effect.  Have this take parameters and return the values